packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/UNSTABLE_EmptyStateButtons.twig

```diff
@@ -0,0 +1,27 @@
+{# API #}
+{%- set _buttonPrimaryText = props.buttonPrimaryText | default(null) -%}
+{%- set _buttonPrimaryUrl = props.buttonPrimaryUrl | default(null) -%}
+{%- set _buttonSecondaryText = props.buttonSecondaryText | default(null) -%}
+{%- set _buttonSecondaryUrl = props.buttonSecondaryUrl | default(null) -%}
+
+{# Class names #}
+{%- set _buttonsClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState__buttons' -%}
+
+<div class="{{ _buttonsClassName }}">
+    {% if _buttonPrimaryText and _buttonPrimaryUrl %}
+        <ButtonLink
+            href="{{ _buttonPrimaryUrl }}"
+            color="primary"
+        >
```

Comment:
No restProps for the buttons?

Reply:
This component will be updated with ActionLayout